decent-dao/fractal-interface
# Usul metadata

## Description
This PR: 
1) Updates the Usul proposal creation process to include a metadata step with fields for "title", "description", and "additional resources" (an external link)
2) Replaces the generated proposal titles from decoded data with the metadata title, if one was provided



## Notes



## Issue / Notion doc (if applicable)
https://github.com/decent-dao/fractal-interface/issues/862


## Testing
1) Create various proposals on a Usul DAO, some providing titles, descriptions, and links, and others not.
2) Verify that the proposals are displayed correctly in the various views, and show the metadata values when appropriate


## Screenshots (if applicable)
